packages/react-devtools-fusebox/src/frontend.d.ts

```diff
@@ -28,22 +28,23 @@ export type Config = {
 export function createBridge(wall: Wall): Bridge;
 export function createStore(bridge: Bridge, config?: Config): Store;
 
-export type Source = {
-  sourceURL: string,
-  line: number,
-  column: number,
-};
+export type ReactFunctionLocation = [
+  string, // function name
+  string, // file name TODO: model nested eval locations as nested arrays
+  number, // enclosing line number
+  number, // enclosing column number
+];
 export type ViewElementSource = (
-  source: Source,
-  symbolicatedSource: Source | null,
+  source: ReactFunctionLocation,
```

Comment:
I don't know if there are any consumers of this in fusebox that needs to be updated. If so, it's probably just the same as the standalone.

https://github.com/facebook/react/pull/33955/files#diff-44b2457a5630ea27596ba5182ef5ec7427b63b6aa2cb4143083eb321e8a12752R153

Reply:
This package is private and the only user is React Native DevTools.

This should be fine,  I will update this next time I upgrade React DevTools for React Native DevTools.